src/developer_config.yaml

```diff
@@ -1,6 +1,6 @@
 global:
   log_to_file: True # Write logs to .log file
-  logging_level: "DEBUG"
+  logging_level: "INFO"
```

Comment:
There's quite a few loggers that are at debug level. Do you want to change some of these to info to keep the saved? See image:
![image](https://github.com/ONSdigital/research-and-development/assets/121172811/c85852c8-b9f2-42de-bf74-f5609c15a366)